<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Factory Business Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="MenuWidget" name="page_menu"/>
      <widget class="StartWidget" name="page_start"/>
      <widget class="ConfWidget" name="page_conf"/>
      <widget class="MainWidget" name="page_main"/>
      <widget class="EndWidget" name="page_end"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_newGame"/>
    <addaction name="action_loadGame"/>
    <addaction name="action_saveGame"/>
    <addaction name="separator"/>
    <addaction name="action_configuration"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <action name="action_newGame">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionLoad_Game">
   <property name="text">
    <string>Load Game</string>
   </property>
  </action>
  <action name="action_loadGame">
   <property name="text">
    <string>&amp;Load Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_configuration">
   <property name="text">
    <string>Configuration...</string>
   </property>
  </action>
  <action name="action_saveGame">
   <property name="text">
    <string>&amp;Save Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StartWidget</class>
   <extends>QWidget</extends>
   <header>startwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfWidget</class>
   <extends>QWidget</extends>
   <header>confwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainWidget</class>
   <extends>QWidget</extends>
   <header>mainwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MenuWidget</class>
   <extends>QWidget</extends>
   <header>menuwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EndWidget</class>
   <extends>QWidget</extends>
   <header>endwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
